Ticket - 25353 <NEW> NK1-BOLFmt=BolFMT15- copy of Xprint2 w' business formlogo at top and units at the bottom

sewa67 opened the pull request:
Please review this change and merge into development branch 
Ticket - oe/rep/bolfibrex.i,oe/rep/bolfibrex.p

sewa67 committed:
Ticket - 25353 <NEW> NK1-BOLFmt=BolFMT15- copy of Xprint2 w' businessformlogo at top and units at the bottom
Ticket - oe/rep/bolfibrex.i,oe/rep/bolfibrex.p

brad-advantzware reviewed (changes requested):
Sewa - This is still just changing the BOLFMTX format for FIBRE.  Before you make anymore changes, we need to clarify spec

sewa67 committed:
Ticket - 25353 <NEW> NK1-BOLFmt=BolFMT15- copy of Xprint2 w' businessformlogo at top and units at the bottom
FIles- oe/rep/bolfrftn.i,oe/rep/bolfrftn.p,oe/rep/bolfrftn2.i,sys/inc/bolform.i,sys/ref/sys-ctrl.i

brad-advantzware reviewed (approved):
THis is a mess but I think the BOLFMTX with "BOLFMTX15" as the char value, should print the Freight BOL as desired.  Approving to test but will need to fail other requirements (including BOLFMT15 for BOLFMT which appears to call a non-existent program).
